src/index.js

```diff
@@ -1,15 +1,45 @@
-import fs from 'fs-extra';
+import fs from 'fs';
 import Plugin from 'broccoli-plugin';
 import md5Hex from 'md5-hex';
 import path from 'path';
 import { default as _logger } from 'heimdalljs-logger';
 import heimdall from 'heimdalljs';
+import FSTree from 'fs-tree-diff';
+import { Promise } from 'rsvp';
 
 // rollup requires this, so old version of node need it
 import 'es6-map/implement';
 
 const logger = _logger('broccoli-rollup');
 
+class Entry {
+  constructor(dest, content) {
+    this.relativePath = dest;
+    this.content = content;
+    this.mode = 0;
+    this.size = content.length;
+    this.checksum = md5Hex(content);
```

Comment:
I believe we should either, not store content `this.content` on line 18, or do the md5hex lazily.